bucket/mangal.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "3.4.1",
+    "homepage": "https://github.com/metafates/mangal",
+    "license": "MIT",
+    "description": "The ultimate CLI manga downloader!",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/metafates/mangal/releases/download/v3.4.1/mangal_3.4.1_Windows_i386.tar.gz",
+            "bin": [
+                "mangal.exe"
+            ],
+            "hash": "99d3aaf0b7f0e9b00d6d2227e7a3aad53a6c467ef4c3840e8214a8ec9c2d5775"
+        },
+        "64bit": {
+            "url": "https://github.com/metafates/mangal/releases/download/v3.4.1/mangal_3.4.1_Windows_x86_64.tar.gz",
+            "bin": [
+                "mangal.exe"
+            ],
+            "hash": "b434d4942e5a6988699213e41d085415ee4d36b28664017608f70f74ba606448"
+        }
+    },
```

```suggestion
    "architecture": {
        "32bit": {
            "url": "https://github.com/metafates/mangal/releases/download/v3.4.1/mangal_3.4.1_Windows_i386.tar.gz",
            "hash": "99d3aaf0b7f0e9b00d6d2227e7a3aad53a6c467ef4c3840e8214a8ec9c2d5775"
        },
        "64bit": {
            "url": "https://github.com/metafates/mangal/releases/download/v3.4.1/mangal_3.4.1_Windows_x86_64.tar.gz",
            "hash": "b434d4942e5a6988699213e41d085415ee4d36b28664017608f70f74ba606448"
        }
    },
    "bin": "mangal.exe",
```

This can be more simple.